tests/integration/test_nitrate.py

```diff
@@ -148,6 +148,36 @@ def test_missing_user_dryrun(self):
                 ["test", "export", "--nitrate", "--debug", "--dry", "."],
                 catch_exceptions=False)
 
+    def test_export_blocked_by_validation(self):
+        os.chdir(self.tmpdir / "validation")
+        fmf_node = Tree(self.tmpdir).find("/validation")
+        with fmf_node as data:
+            data['test'] = 'echo hello world'
+        runner = CliRunner()
+        with self.assertRaises(ConvertError) as error:
+            self.runner_output = runner.invoke(
+                tmt.cli.main,
+                ["test", "export", "--nitrate", "--debug", "--dry", "."],
+                catch_exceptions=False)
+        self.assertIn('Not committed changes', str(error.exception))
+
+    def test_export_forced_validation(self):
+        os.chdir(self.tmpdir / "validation")
+        fmf_node = Tree(self.tmpdir).find("/validation")
+        with fmf_node as data:
+            data['extra-nitrate'] = 'TC#599605'
+
+        runner = CliRunner()
+
+        self.runner_output = runner.invoke(
+            tmt.cli.main,
+            ["test", "export", "--nitrate", "--debug", "--force-git-validation", "."],
+            catch_exceptions=False)
+
+        self.assertIn(
+            "Exporting regardless Not committed changes",
```

Comment:
```suggestion
            "Exporting regardless uncommitted changes",
```

The one-word form looks just a bit more straightforward?